savings-account.rb

```diff
@@ -0,0 +1,16 @@
+module Bank
+  class SavingsAccount < Account
```

Comment:
Nice! The minimal amount of code you have here shows that you used the inheritance concept quite well